src/WikiService.Business/Commands/Rubric/Edit/EditRubricHandler.cs

```diff
@@ -27,17 +28,56 @@ private Task<DbRubric> GetAsync(Guid rubricId, CancellationToken ct)
       return _provider.Rubrics.FirstOrDefaultAsync(x => x.Id == rubricId, ct);
     }
 
+    private async Task ChangeActivationAsync(Guid rubricId, bool isActivate)
```

Comment:
may be ChangeActivityAsync or ActivateAsync?

Reply:
done